Kickstarter-iOS/DataSources/SettingsAccountDataSource.swift

```diff
@@ -6,7 +6,7 @@ final class SettingsAccountDataSource: ValueCellDataSource {
     clearValues()
     SettingsAccountSectionType.allCases
       .forEach { section -> Void in
-      let values = section.cellRowsForSection.map { SettingsCellValue(user: nil, cellType: $0) }
+      let values = section.cellRowsForSection.map { SettingsCellValue(cellType: $0) }
```

Comment:
If you're in that point-free mood 🤤

Reply:
If there was suggestion here as well I could have batched these on Github...this way I went to Xcode and did these 2 changes in IDE.

Reply:
Ugh, also I believe you can't point free something that has multiple optional params :/